example/src/main/java/com/stripe/example/activity/LauncherActivity.kt

```diff
@@ -65,7 +65,9 @@ class LauncherActivity : AppCompatActivity() {
             Item(activity.getString(R.string.sofort_example),
                 SofortPaymentMethodActivity::class.java),
             Item(activity.getString(R.string.card_brands),
-                CardBrandsActivity::class.java)
+                CardBrandsActivity::class.java),
+            Item("Simple Redirect PaymentMethod Confirmation Example",
+                SimpleRedirectPaymentMethodActivity::class.java)
```

Comment:
Maybe "Simple PaymentMethod Confirmation Example" is enough?

Reply:
Sure. Names are hard